erpnext/tests/test_zform_loads.py

```diff
@@ -0,0 +1,29 @@
+""" dumb test to check all function calls on known form loads """
```

Comment:
![](https://decider.com/wp-content/uploads/2019/04/daenerys-mean-girl-face-game-of-thrones.jpg?quality=90&strip=all&w=1249&h=856&crop=1)